src/components/dialogs/network-modifications/tabular-modification/tabular-modification-utils.ts

```diff
@@ -112,42 +111,52 @@ export const formatModification = (modification: Modification) => {
     return rest;
 };
 
-export const convertValueFromBackToFront = (key: string, value: { value: string | number }) => {
+export const getEquipmentTypeFromModificationType = (type: string) => {
+    return Object.keys(TABULAR_MODIFICATION_TYPES).find((key) => TABULAR_MODIFICATION_TYPES[key] === type);
+};
+
+export const styles = {
+    grid: { height: 500, width: '100%' },
+};
+
+export const convertInputValue = (key: string, value: { value: string | number }) => {
     switch (key) {
         case EQUIPMENT_ID:
             return value;
         case G:
+            return convertInputValues(FieldType.G, value?.value);
         case B:
+            return convertInputValues(FieldType.B, value?.value);
         case G1:
+            return convertInputValues(FieldType.G1, value?.value);
         case G2:
+            return convertInputValues(FieldType.G2, value?.value);
         case B1:
+            return convertInputValues(FieldType.B1, value?.value);
         case B2:
-            return unitToMicroUnit(value?.value);
+            return convertInputValues(FieldType.B2, value?.value);
```

Comment:
to thinking about MAX_SUSCEPTANCE, in IHM is S and IIDM is MicroS

Also, the csv file the value edited for MAX_SUSCEPTANCE is in S or MicroS ?

Reply:
It is planned to handle the fix for labels for all the IHM of gridstudy in another PR (we have already another US to fix this)